README.md

```diff
@@ -2,16 +2,16 @@
 
 [![Build Status](https://travis-ci.org/igneus/church-calendar-api.svg?branch=master)](https://travis-ci.org/igneus/church-calendar-api)
 
-Browse Roman Catholic liturgical calendar through a web interface
-or obtain it's data in JSON format through an API.
+Browse the Roman Catholic liturgical calendar through a web interface
+or obtain its data in JSON format through an API.
 
 Powered by
 [calendarium-romanum][caro] and
 grape.
 
-## Explore it on-line
+## Explore it online
```

Comment:
This is only a minor point. Both are legitimate - that said, at least here in Canada, we seldom see it written as 'on-line'

Reply:
I know it's rare nowadays, but I prefer the (now perceived as mildly archaic) dashed form.